tests/test_check_function.py

```diff
@@ -221,26 +222,29 @@ def test_method_1():
     )
     helper.passes(s.check_function("df.groupby").check_args(0).has_equal_value())
     helper.passes(s.check_function("df.groupby.sum", signature=False))
-    from pythonwhat.signatures import sig_from_obj
-    import pandas as pd
 
+    import inspect
+    from inspect import Parameter as param
+    manual_sig = inspect.Signature([
+        param("x", param.POSITIONAL_OR_KEYWORD, default=None),
+        param("axis", param.POSITIONAL_OR_KEYWORD, default=None)
+    ])
     helper.passes(
-        s.check_function("df.groupby.sum", signature=sig_from_obj(pd.Series.sum))
```

Comment:
In the end you couldn't get sig_from_obj to work on this function?

Reply:
Supporting it should be possible, but it could be a tech debt / ice box ticket, depending on adoption of this version.

Reply:
Definitely, however since this test never really worked and wasn't checking if  sig_from_obj was working on a class function (that has self as first argument), I though it was more a feature/tech debt than a bug introduced by the upgrade. We will be able to see how much courses/exercises are affected when upgrading to this version so that should tell us the value of implementing this later.